website/pages/api-docs/features/filtering.mdx

```diff
@@ -9,83 +9,48 @@ description: |-
 
 # Filtering
 
-A filter expression is used to refine a data query for some API listing endpoints as notated in the individual API documentation.
-Filtering will be executed on the Consul server before data is returned, reducing the network load. To pass a
-filter expression to Consul, with a data query, use the `filter` parameter.
+Filter expressions refine data queries for some API listing endpoints, as notated in the individual API documentation.
+
+To create a filter expression, you will write one or more expressions. Each expression has a matching operators composed of selectors and values.
```

Comment:
The matching operators are the `==`, `!=`, `is`, etc. They don't contain the selectors and values themselves.

```suggestion
To create a filter expression, you will write one or more expressions. Each expression has a matching operators composed with selectors and values.
```